jacd.c

```diff
@@ -217,6 +217,7 @@ static int open_socket(const char *device, uint64_t name)
 			.addr = J1939_IDLE_ADDR,
 			.pgn = 0x0ee00,
 		},
+		.can_ifindex = if_nametoindex(s.intf),
```

Comment:
This is a direct consequence of not allowing bind to any device ("3cb6a1204598 j1939: socket: bind() cleanups"), i.e. `can_ifindex == 0`. This means before this change `jacd` would receive from any interface that was opened by another program.

Reply:
Sounds reasonable, what should we do here?

Reply:
Just wanted to explain why this is necessary now and that I broke it with my commit.

Reply:
Ah, got it. I will add that information to the commit message.